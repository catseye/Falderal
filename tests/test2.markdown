Falderal Test 2
---------------

Since no functionality is specified for these tests,
`py-falderal` should exit with an exception.

    -> Functionality "Cat" is implemented by
    -> shell command "python cat.py"

Cat cats.

    | meow
    = meow

    | purr
    | prrr
    | prreow
    = purr
    = prrr
    = prreow
