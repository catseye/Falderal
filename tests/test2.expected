FalderalSyntaxError: line 13: functionality under test not specified
